src/main/scala/mesosphere/marathon/core/launchqueue/impl/LaunchQueueActor.scala

```diff
@@ -182,27 +187,28 @@ private[impl] class LaunchQueueActor(
       }
 
     case Add(app, count) =>
-      launchers.get(app.id) match {
-        case None =>
-          import context.dispatcher
-          val actorRef = createAppTaskLauncher(app, count)
-          val eventualCount: Future[QueuedInstanceInfo] =
-            (actorRef ? TaskLauncherActor.GetCount).mapTo[QueuedInstanceInfo]
-          eventualCount.map(_ => Done).pipeTo(sender())
+      import context.dispatcher
 
-        case Some(actorRef) =>
-          import context.dispatcher
-          val eventualCount: Future[QueuedInstanceInfo] =
-            (actorRef ? TaskLauncherActor.AddInstances(app, count)).mapTo[QueuedInstanceInfo]
-          eventualCount.map(_ => Done).pipeTo(sender())
-      }
+      async {
+        val startInstances: Iterable[Future[Done]] = 0.until(count).toIterable.map { _ =>
+          val newInstance: Instance = Instance.Scheduled(app, Instance.Id.forRunSpec(app.id))
+          logger.info(s"Adding instance ${newInstance.instanceId}")
+          instanceTracker.schedule(newInstance)
```

Comment:
This is the important change. Scheduling an instance becomes a simple `create`. I think we can remove the queue all together.